synapseclient/models/mixins/asynchronous_job.py

```diff
@@ -151,129 +195,156 @@ def fill_from_dict(self, async_job_status: dict) -> "AsynchronousJobStatus":
         return self
 
 
-class AsynchronousJob:
+async def send_job_and_wait_async(
```

Comment:
should these be in a separate script?

Reply:
Good point. This would probably make sense to live within a py script at `synapseclient.api`

Reply:
I can do this in the main feature branch

Reply:
@BryanFauble Actually, maybe that isn't possible. If we move `get_job_async` we need to move the AsynchronousJobStatus and AsynchronousJobState classes to avoid a circular import issue